### ISSUE TITLE
FFprobe output empty

### ISSUE THREAD

**Issue by antoniobrandao** [fluent-ffmpeg/node-fluent-ffmpeg#447]
I'm using ffprobe and the result if always like this:

	{ filename: '5da36f5c4b07839ec905b588832cc62b.mp4',
	  filepath: '/Users/antoniobrandao/WEBROOT/Electron.app/Contents/Resources/default_app/builds/development/resources/upload/images',
	  fileext: '.mp4',
	  file: '/Users/antoniobrandao/WEBROOT/Electron.app/Contents/Resources/default_app/builds/development/resources/upload/images/5da36f5c4b07839ec905b588832cc62b.mp4',
	  probe_time: 94,
	  streams: null,
	  format: null,
	  metadata: undefined }

I am using a local copy of ffprobe version 2.6.3 on OSX yosemite.

This happens with any video I probe. The command invoking the probe is, in nodejs:

spawn(module.exports.ffmpeg_path, ['-show_streams', '-show_format', '-loglevel', 'warning', file])

Any ideas ?

**Comment by njoyard** [comment]
What is the result on command line ? Are you calling ffmpeg or ffprobe (your code above is not clear on this point) ?

**Comment by antoniobrandao** [comment]
Sorry for the missing points. I am invoking ffprobe, and it is a command line command spawned by nodejs. 

It's the Node.js module https://www.npmjs.com/package/node-ffprobe at work. It simply spawns a command line command.

This has worked before in the same machine, but with ffmpeg installed in my system via homebrew. 

But I removed ffmpeg from my system and I'm trying to use it locally.

Now I'm using a local copy of the ffmpeg binaries (for each OS), and I am directly invoking the ffprobe binary. The file path is correct and the invoked command is the same.

I confirmed the provided path to the ffprobe binary is being found by the node.js script, thus executing it and I get that output in the console which only happens because ffprobe is in fact executed.

Could it be that the problem is me invoking the executable without ffmpeg being actually installed in my system? Does it have to?

I am using the original, untouched file tree structure of the ffmpeg binaries package.

FFMpeg version is 2.6.3

And here is a screenshot of the package's tree structure https://www.dropbox.com/s/xu3iyht111krf3u/Screenshot%202015-09-14%2022.47.25.png?dl=0
